x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/constants.ts

```diff
@@ -242,6 +242,13 @@ export const ROLES_DISABLED_NOTE = i18n.translate(
   }
 );
 
+export const RBAC_BUTTON_DISABLED_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.roleMapping.rbacButtonDisabledLabel',
+  {
+    defaultMessage: 'Enabling RBAC can be performed by a superadmin user.',
```

Comment:
```suggestion
    defaultMessage: 'Enabling RBAC can be performed by a superuser.',
```
I haven't found "superadmin" in the docs anywhere. I think "superuser" will be better here as this is the terminology used in docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/built-in-users.html